frontend/src/components/Board/Board.tsx

```diff
@@ -1,7 +1,339 @@
-import React from 'react';
+import { useState } from 'react';
+// import { AccountCircle } from "@mui/icons-material";
```

Comment:
Is this import gonna be used?

Reply:
was going to for the member icons, but left it out for now.